resources/views/users/connected.blade.php

```diff
@@ -0,0 +1,18 @@
+@extends('layout')
+
+@section('title')
+    You've sent a connection request!
+@stop
+
+@section('content')
+
+    @if($connection->user_id == $connection->owner_id)
+        You connected with yourself! Doesn't it feel good to know the real you?
+    @else
+        You connected with {{App\User::findOrFail($connection->user_id)->name}}
```

Comment:
Rule: Unless you have a specific reason to, never perform database queries in a view template. It almost always leads to extremely slow and brittle code.

Get data in a controller.

Reply:
fixed